optimum/habana/transformers/trainer.py

```diff
@@ -1318,72 +1309,6 @@ def _load_rng_state(self, checkpoint):
                         "\nThis won't yield the same results as if the training had not been interrupted."
                     )
 
-    def _save_checkpoint(self, model, trial, metrics=None):
```

Comment:
There is a call to _save_checkpoint on line 1267. Wondering if the function got moved elsewhere?

Reply:
The function is still in use and remains unchanged from the superclass `Trainer` in [`transformers`](https://github.com/huggingface/transformers/blob/53fad641cfdb5105e2470bcf3ef17ea8e25cc300/src/transformers/trainer.py#L2995) . Therefore, there is no need to rewrite it in this class.